machine/util/log.py

```diff
@@ -167,34 +166,21 @@ def plot_metric(self, metric_name, restrict_model=lambda x: True,
                                 steps, log.data[dataset][metric_name][:eor])
                             ax.plot(steps, data,
                                     color_group(name, dataset),
-                                    label=label + label_name, linewidth=3.0)
+                                    label=label + label_name, linewidth=3.0, **line_kwargs)
                         else:
                             ax.plot(steps,
                                     log.data[dataset][metric_name][:eor],
-                                    label=label + label_name)
+                                    label=label + label_name, **line_kwargs)
                         ax.tick_params(
                             axis='both', which='major', labelsize=20)
                         plt.xlabel("Epochs", fontsize=24)
-                        plt.ylabel("Sequence Accuracy", fontsize=24)
+                        plt.ylabel(metric_name, fontsize=24)
```

Comment:
Always putting "Sequence Accuracy" is not really the best option, I think "metric_name" isn't either, because this is often not really a human-readable sequence (e.g. seqacc). Can we maybe think of another solution for this?

Reply:
Instinctively maybe adding another named argument like `y_label=None` to the function signature, if it is `None` `metric_name` will be used, otherwise `y_label`?

Reply:
good idea